Header header
geometry_msgs/PoseStamped x
geometry_msgs/PoseStamped x_desi
geometry_msgs/PoseStamped x_desi_filtered
geometry_msgs/Twist x_err
geometry_msgs/Twist xd
geometry_msgs/Twist xd_desi
geometry_msgs/Wrench F
float64[] q_proxy
float64[] qd_pose
float64[] qd_posture
float64[] qd_posture_raw
float64[] qd_desi
float64[] tau
std_msgs/Float64MultiArray J
std_msgs/Float64MultiArray N

# Environmental stiffness
float64 df
float64 dx
float64 Df
float64 Dx
float64 stiffness
float64 compliance
